fcli-core/fcli-common/src/main/java/com/fortify/cli/common/session/cli/cmd/AbstractSessionLogoutCommand.java

```diff
@@ -30,8 +30,7 @@ public JsonNode getJsonNode() {
         var sessionHelper = getSessionHelper();
         if ( sessionHelper.exists(sessionName) ) {
         	result = sessionHelper.sessionSummaryAsObjectNode(sessionName);
-            logout(sessionName, sessionHelper.get(sessionName, true));
-            // TODO Optionally delete all variables
+            logout(sessionName, sessionHelper.get(sessionName, false));
```

Comment:
Probably better to use a try/catch/finally block, to always invoke `getSessionHelper().destroy(sessionName)` even if the `logout` method throws an exception. TBD: throw exception if logout method throws exception, or log a warning?